src/cli/arch/build.py

```diff
@@ -0,0 +1,11 @@
+import sys
+import numpy as np
+
+from os import  system
+
+if __name__ == '__main__':
+
+    sp = np.array(sys.path)
+    np.save("/usr/lib/Auth/Facerec/deps_path.npy", sp)
+    system("sudo python3 /usr/lib/Auth/Facerec/cli_info.py")
```

Comment:
Can't we use the build.py that is already there?

Reply:
We do not need common-auth for ArchLinux instead, it should be done manually by adding or removing  `auth      sufficient   pam_python.so /lib/Auth/Facerec/pam_ptn.py` from `/etc/pam.d/sudo` to enable and disable respectively.